GetLatestMigrationsStatus/handler.ts

```diff
@@ -65,21 +86,51 @@ export const createSqlStatus = (dbConfig: IDecodableConfigPostgreSQL) => (
     .as("t")
     .toQuery() as NonEmptyString;
 
-export const createHandler = (): Handler => (): ReturnType<Handler> =>
+export const getLatestMigrationByOrganizationFiscalCode = (
+  config: IConfig,
+  connect: Pool
+) => (
+  organizationFiscalCode: OrganizationFiscalCode
+): TE.TaskEither<IDbError, LatestMigrationResultSet> =>
+  pipe(
+    createSqlStatus(config)(organizationFiscalCode, SubscriptionStatus.INITAL),
+    sql => queryDataTable(connect, sql),
+    TE.mapLeft(flow(toPostgreSQLErrorMessage, toPostgreSQLError))
+  );
+
+export const processResponseFromLatestMigrationResultSet = (
+  resultSet: LatestMigrationResultSet
+): TE.TaskEither<
+  IResponseErrorInternal | IResponseErrorNotFound,
+  IResponseSuccessJson<MigrationsStatus>
+> =>
+  pipe(
+    resultSet,
+    LatestMigrationResultSet.decode,
```

Comment:
Why the need to decode an object of type `LatestMigrationResultSet` with a codec of the very same type? You either:
* don't trust the function signature (why?)
* don't need such decode